src/main/java/org/fcrepo/migration/foxml11/NamespacePrefixMapper.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * 
+ */
```

Comment:
Is this empty javadoc comment required by checkstyle or our javadoc compilation?  If not, we might as well remove it.